config.sh

```diff
@@ -17,5 +17,7 @@ function run_tests {
     # Runs tests on installed distribution from an empty directory
     export PYTHONHASHSEED=$(python -c 'import random; print(random.randint(1, 4294967295))')
     python -c 'import pandas; pandas.show_versions()'
-    python -c 'import pandas; pandas.test(extra_args=["--skip-slow", "--skip-network", "--skip-db", "-n=2"])'
+    # --deselect for 0.24.x
```

Comment:
Alternatively, we could write a patch and apply it before running the tests. I'm not sure which is preferred. I'm also not sure if the patch would affect the pandas version number picked up by versioner (IIRC that's set at this point, but I may be wrong).

Reply:
I would really prefer not to change this at all. these need to be fixed in pandas 0.24.x branch and then just re-run this.

Reply:
some more patches need to be backported.

Reply:
Would that require a new tag?

Reply:
Certainly, we should do the backports like https://github.com/pandas-dev/pandas/pull/25186 anyway.

Reply:
you can overwrite the tag if need be as it hasn't been released

Reply:
I would vote to just skip the tests here, as Tom is doing. It has already been released on github. Making a new tag / release is easy, but not sure it is worth it for skipping a test.

Reply:
what???? why was this put on github before this??? that is crazy